lib/FrontendTool/FrontendTool.cpp

```diff
@@ -992,6 +992,28 @@ static void performEndOfPipelineActions(CompilerInstance &Instance) {
     }
   }
 
+  if (Invocation.getLangOptions()
+          .EnableExperimentalEagerClangModuleDiagnostics) {
+    class ForgetfulConsumer : public VisibleDeclConsumer {
```

Comment:
Can you please add a comment here? 

Maybe a better name is eager consumer? The point of this class is to just import everything, right?